cgmanifest.json

```diff
@@ -19,6 +19,116 @@
         }
       },
       "DevelopmentDependency": true
+    },
+    {
+      "Component": {
+        "Type": "other",
+        "Other": {
+          "Name": "cmocka",
+          "Version": "1.1.5-1",
+          "DownloadUrl": "https://gitlab.com/cmocka/cmocka"
+        }
+      },
+      "DevelopmentDependency": false
```

Comment:
Are we including parts of CMocka outside of the tests?  Otherwise I'd expect this one is true that its a dev only dependency